examples/08_interpolation_join.py

```diff
@@ -0,0 +1,179 @@
+"""
+Interpolation join: infer missing rows when joining two tables
+==============================================================
+
+We illustrate the ``InterpolationJoiner``, which is a type of join where values from the second table are inferred with machine-learning, rather than looked up in the table.
```

Comment:
Best is to link the name to the function (see other examples):
```suggestion
We illustrate the :class:`~skrub.InterpolationJoin`, which is a type of join where values from the second table are inferred with machine-learning, rather than looked up in the table.
```